src/components/HeaderItems/DocHandlerButtons.jsx

```diff
@@ -1,6 +1,12 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+
 import Button from "../shared/Button";
+import AddDocumentModal from "../Modals/AddDocumentModal";
```

Comment:
앱 전반적으로 생성 시에 `create`라는 표현을 주로 사용하고 있어서, 큰 의미 차이는 없겠지만 `CreateDocumentModal`로 컴포넌트 이름 수정하는게 통일성 있을 것 같습니다!
```suggestion
import CreateDocumentModal from "../Modals/CreateDocumentModal";
```

Reply:
초기 UX를 고려하면서 만든 figma 목업에서 아래와 같은 이유로 create와 add로 나뉘었습니다!!

- database는 1개 단위.
- document는 수십 수백 이상 단위.
- 유저가 두 모달을 헷갈리지 않고 쉽게 구분할 수 있어야 함.

개발자 관점에서는 분명 둘다 create에 해당하지만, 위와 같은 이유를 생각해서 create/add로 용어를 구분해주어 유저가 이용하기 편리하도록 배려하면 어떨까 합니다!!

Reply:
유진님 말씀대로 database는 **생성**, document는 정해진 field에 따라 **추가**하는 것이니 create/add 용어 구분이 좋아보입니다 👍

백엔드에서는 `createDatabase`, `createDocument` 함수 네이밍을 사용하고 있는데, 추후에 고쳐놓도록 하겠습니다 😁